src/Admin/CitizenProjectAdmin.php

```diff
@@ -154,6 +154,8 @@ protected function configureFormFields(FormMapper $formMapper)
                 ])
                 ->add('proposedSolution', null, [
                     'label' => 'Solution du problème',
+                    'filter_emojis' => true,
+                    'attr' => ['class' => 'ck-editor'],
```

Comment:
missing `'purify_html' => true`?

Reply:
it's a WISIWIG not purify HTML it's needed no ?

Reply:
Never trust the client.